_posts/2020-06-20-Basic-Querying.md

```diff
@@ -0,0 +1,302 @@
+---
+layout: post
+date: 2020-06-20 18:00:00 +010
```

Comment:
I believe the time zone (+010) is incorrectly set. I had to remove it for the system to build and render the page. :)